src/com/sun/ts/tests/jaxrs/common/provider/StringBeanRuntimeDelegate.java

```diff
@@ -17,12 +17,18 @@
 package com.sun.ts.tests.jaxrs.common.provider;
 
 import jakarta.ws.rs.core.Application;
-import jakarta.ws.rs.core.Link.Builder;
+import jakarta.ws.rs.core.Link;
+import jakarta.ws.rs.core.EntityPart;
 import jakarta.ws.rs.core.Response.ResponseBuilder;
 import jakarta.ws.rs.core.UriBuilder;
 import jakarta.ws.rs.core.Variant.VariantListBuilder;
 import jakarta.ws.rs.ext.RuntimeDelegate;
 
+import java.util.concurrent.CompletionStage;
+
+import jakarta.ws.rs.SeBootstrap;
```

Comment:
What is this test verifying?  In the new REST TCK, I see this class being referenced from other tests but I don't see any references to StringBeanRuntimeDelegate in this pull request.

Reply:
You are right, it is not used. There are more files under src/com/sun/ts/tests/jaxrs/common that can be removed. I will make that change.